File: app/src/main/java/app/islammedia/halaqatalquran/Database/Halaqa.java
package app.islammedia.halaqatalquran.Database;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity
public class Halaqa {
    @PrimaryKey(autoGenerate = true)
    int hid;

    @ColumnInfo(name = "info")
    String info;

    @ColumnInfo(name = "name")
    public String name;

    @ColumnInfo(name = "halaqa_time")
    String time;

    public Halaqa(String name, String time, String info){
        this.name=name;
        this.info=info;
        this.time=time;
    }
}

File: app/src/main/java/app/islammedia/halaqatalquran/database/entities/Halaqa.kt
package app.islammedia.halaqatalquran.database.entities

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey
import app.islammedia.halaqatalquran.database.helpers.HalaqaDate

@Entity
class Halaqa(@kotlin.jvm.JvmField @ColumnInfo(name = "name") var name: String,
             @kotlin.jvm.JvmField @ColumnInfo(name = "info") var info: String?,
             @kotlin.jvm.JvmField @ColumnInfo(name = "halaqa_dates") var halaqaDates: MutableList<HalaqaDate?>?,
             @kotlin.jvm.JvmField @PrimaryKey var halaqaID: String)